client/src/components/post/Posts.js

```diff
@@ -14,30 +15,40 @@ const Posts = () => {
   const postContext = useContext(PostContext)
   const { posts } = postContext
 
+  const authContext = useContext(AuthContext)
+  const { loggedInUser } = authContext
+
   return (
     <div>
       {posts.map((post) => (
         <div className='card text-dark bg-light mb-3' key={post.id}>
-          <div className='card-header d-flex justify-content-between'>
-            <div className='d-flex align-items-center '>
-              <img
-                src={profilePicture}
-                alt='profile'
-                className='post-profile-picture border border-secondary rounded mr-3'
-                id='profile-picture'
-              />
-              <Link to={`/user/${post.user_id}`}>
-                <h6 className='text-dark'>
-                  {post.first_name} {post.last_name}
-                </h6>
-              </Link>
-            </div>
-            <div className='d-flex align-items-end'>
-              {convertDate(post.created_at)}
+          <div className='card-header'>
+            <div className='d-flex justify-content-between'>
+              <div className='d-flex align-items-center '>
+                <img
+                  src={profilePicture}
+                  alt='profile'
+                  className='post-profile-picture border border-secondary rounded mr-3'
+                  id='profile-picture'
+                />
+                <Link to={`/user/${post.user_id}`}>
+                  <h6 className='text-dark'>
+                    {post.first_name} {post.last_name}
```

Comment:
mm for consistency's sake, in the future, line up your api and ui to use the same casing type. It doesn't really matter but its nice to have everything either snake case or camel case and i know javascript people like to use camelCase

Reply:
Database as well? I'd probably prefer camelCase, but I guess I'm use to seeing snake case in sql

Reply:
db you have to use kebab case, i dont think the db respects casing